### ISSUE TITLE
Generating a Single Table with Patients and Conditions

### ISSUE THREAD

**Issue by abdulzeedo** [synthetichealth/synthea#763]
Hi guys,

I would like to generate a single table with only the patient details i.e. DoB, gender, zip, etc and the description field of the Conditions table. I have read through the wiki but I couldn't find a suitable configuration for it. 

Is this possible? 

Another thing is that I would like to get only disorders rather than things like `Body mass index 30+ -   obesity (finding)` in the description field of the Conditions table. 

The application is relevant to data anonymisation.

**Comment by jawalonoski** [comment]
@abdulzeedo there is no out of the box configuration to produce that. Your best choice is to post-process the data at this point.

You could join the `patients.csv` with the `conditions.csv` if you loaded them into a relational database (for example).

Joining `patient.id` with `conditions.patient` should do the trick. Theoretically you could add a where-clause for disorders. Something like `conditions.description ilike %disorder%` -- although my SQL is rusty so that probably doesn't work exactly, but you get the idea.

**Comment by abdulzeedo** [comment]
@jawalonoski thanks for the reply. Yes, seems like it's the only option. 
Another question, the relationship between patients and conditions is a 1-to-many, right?

**Comment by jawalonoski** [comment]
> Another question, the relationship between patients and conditions is a 1-to-many, right?

Yes, although a patient might not have any conditions (if they are healthy).
